src/libcfitsio.jl

```diff
@@ -56,6 +56,8 @@ isdefined(Base, :__precompile__) && __precompile__()
 
 module Libcfitsio
 
+import FITSIO: fits_assert_isascii
+
```

Comment:
This is getting a bit nitpicky, but I think it was better the way you had it, with `fits_assert_isascii` defined in Libcfitsio. Mainly just because we already have some non-exported stuff there that is used by `FITSIO` and follows that pattern (see https://github.com/JuliaAstro/FITSIO.jl/blob/d3d9af24359f3ec90f79866dad2c33736bb50fcd/src/FITSIO.jl#L41-L47). Also it seems cleaner for submodules to not depend on their parent modules, if possible.

Reply:
> Also it seems cleaner for submodules to not depend on their parent modules, if possible.

To be honest, I'd have said the other way round.  However I never played much with submodules, so don't know the subtleties of this.

Reply:
Yeah I might be wrong about that. We can probably agree that it is better to have a one-way rather than bidirectional dependency though!

Reply:
Yes, I agree ;-)